app/presenters/session_timeout_modal_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class SessionTimeoutModalPresenter
+  def initialize(user_fully_authenticated:)
+    @user_fully_authenticated = user_fully_authenticated
+  end
+
+  def locale_scope
```

Comment:
supernit, I feel like `scope` or `translation_scope` is more precise than `locale_scope`? To me, the locale is the language (`en-US`, `en`, etc) and the i18n keys are independent of that (their values are different)

Reply:
That makes sense to me, I can change it to `translation_scope`.

Reply:
Renamed in 857e81fa0.